docs/02_stratum0_client.md

```diff
@@ -1,136 +1,290 @@
+In order to get started with CernVM-FS, the first thing you need is a Stratum 0 server.
 
-## Set up the Stratum 0
+The Stratum 0 is the *central server* that hosts your repositories and makes it available to other systems.
+**There can be only one Stratum 0 server for each CernVM-FS repository**, and from a security perspective it is recommended to restrict the access to this system.
+We will look more into that later.
 
-### Requirements
-Due to the scalable design of CernVM-FS, the host of your Stratum 0 server does not need a lot of resources in terms of CPU cores and memory; just a few cores and a few gigabytes of memory should suffice. Besides this, you need plenty of space to store the contents of your repository. By default, CernVM-FS uses `/var/spool` as scratch space while adding new files to the repository, and `/srv/cvmfs` as central repository storage location, but these locations can be modified in the CernVM-FS server settings later on.
+For now, we are going to set up a Stratum 0, create a repository, and access it by connecting from a client machine directly to the Stratum 0 server.
 
-Furthermore, several (popular) Linux distributions are supported, see [these requirements](https://github.com/cvmfs-contrib/cvmfs-tutorial-2021/wiki/Notes#stratum-1--proxies) for a full list. We will only focus on CentOS in this tutorial.
+!!! warning
+    **Directly connecting to the Stratum 0 is** ***not*** **recommended**, but it is a good way to get started.
 
-CernVM-FS also offers support for hosting the repository contents in S3 compatible storage, but for this tutorial we will focus on storing the files locally. For this we need an Apache server on the host, and port 80 should be open.
+    In the [next hands-on part of this tutorial](03_stratum1_proxies.md) we will remedy this by setting up a Stratum 1
+    replica server and squid proxy, and accessing the repository that way instead.
 
-### Installation
-The installation of CernVM-FS is simple and only requires some packages to be installed. You can easily do this by adding the CernVM-FS repository and install the packages through your package manager:
+## 2.1 Setting up the Stratum 0
+
+### 2.1.1 Requirements
+
+#### Resources
+
+Due to the scalable design of CernVM-FS, the Stratum 0 server does not need a lot of resources in terms of CPU cores and memory;
+just a few cores and a couple gigabytes of memory is sufficient.
+
+Besides this, you do need sufficient storage space to store the contents of your repository. By default, CernVM-FS uses `/var/spool` as scratch space
+while adding new files to the repository, and `/srv/cvmfs` as central repository storage location, but these locations can be modified in the
+CernVM-FS server configuration settings later on.
+
+#### Operating system
+
+Several (popular) Linux distributions are supported by CernVM-FS,
+see [these requirements](https://github.com/cvmfs-contrib/cvmfs-tutorial-2021/wiki/Notes#stratum-1--proxies) for a full list.
```

Comment:
Don't know how this url ended up here (it was already like that in my version, apparently), but it somehow points to the wiki of this github repo.
```suggestion
see [the Getting Started page of the CernVM-FS documentation](https://cvmfs.readthedocs.io/en/stable/cpt-quickstart.html#getting-the-software) for a full list.
```